kits/cpp/src/lux/unit.cpp

```diff
@@ -7,6 +7,13 @@
 #include "lux/observation.hpp"
 
 namespace lux {
+
+    int64_t Unit::actionQueueCost(const Observation &obs) const {
```

Comment:
The signature of this (`int64_t actionQueueCost(const Observation &obs) const;`) must be added to the `Unit` struct in `unit.hpp`.
Otherwise this will not compile.